templates/single-meetings.php

```diff
@@ -95,7 +80,7 @@ function add_hour_to_start_time($start) {
 						<?php _e($tsml_meeting_attendance_options[$meeting->attendance_option], "12-step-meeting-list") ?>
 					</div>
 					<br />
-					<?php echo tsml_link(get_post_type_archive_link('tsml_meeting'), '<i class="glyphicon glyphicon-chevron-right"></i> ' . __('Back to Meetings', '12-step-meeting-list-feedback-enhancement'), 'tsml_meeting') ?>
+					<?php echo tsml_link(get_post_type_archive_link('tsml_meeting'), '<i class="glyphicon glyphicon-chevron-right"></i> ' . __('Back to Meetings', '12-step-meeting-list'), 'tsml_meeting') ?>
```

Comment:
Ditto. I am not smart enough in the intricacies of Wordpress to understand the impacts of a change like this.

Reply:
Right. Will fix...